### PR TITLE
New stuff

### PR DESCRIPTION
**Pull request by GeoffreyWesthoff**
- pls tweet will now have data from the user instead of the big meme man
- added support for proxy servers

### CODE REVIEW

File: utils/http.py [hunk 1]
```diff
@@ -2,14 +2,24 @@
...
+import json
+
+config = json.load(open('config.json'))
 
 
 def get_image(url):
     try:
-        return Image.open(BytesIO(requests.get(url, stream=True).content))
+        if 'proxy_url' in config:
+            return Image.open(BytesIO(requests.get(config['proxy_url'] + '?url=' + url,
```
Comment (AndyTempel) [inline utils/http.py]: I think it's better to use the in-built GET parameter generator in requests. `requests.get(config['proxy_url'], params={'url': url})`

File: utils/http.py [hunk 2]
```diff
@@ -2,14 +2,24 @@
...
+        else:
+            return Image.open(BytesIO(requests.get(url, stream=True).content))
     except OSError:
         raise TypeError('An invalid image was provided! Check the URL and try again.')
 
 
 def get_image_raw(url):
-    return requests.get(url, stream=True).content
+    if 'proxy_url' in config:
+        return requests.get(config['proxy_url'] + '?url=' + url, stream=True).content
```
Comment (AndyTempel) [inline utils/http.py]: Same here

**Review by AndyTempel — CHANGES_REQUESTED** [review]

**Comment by GeoffreyWesthoff** [comment]
Good suggestion. It works fine as it is but perhaps this will help not having to deal with funky edge cases.

File: utils/http.py [hunk 3]
```diff
@@ -10,8 +10,7 @@
 def get_image(url):
     try:
         if 'proxy_url' in config:
-            return Image.open(BytesIO(requests.get(config['proxy_url'] + '?url=' + url,
-                                                   stream=True).content))
+            return Image.open(BytesIO(requests.get(config['proxy_url'],params={'url': url}, stream=True).content))
```
Comment (AndyTempel) [inline utils/http.py]: `['proxy_url'],params={'url':` lint bitte schön

**Review by AndyTempel — CHANGES_REQUESTED** [review]

File: utils/http.py (hunk 3 above)
Comment (AndyTempel) [reply utils/http.py]: @Devoxin `run_tests.py` when? :^)

Comment (GeoffreyWesthoff) [reply utils/http.py]: Who even cares about linting :mmlol:

Comment (devoxin) [reply utils/http.py]: \>:(

**Review by devoxin — APPROVED** [review]

**Pull request merged**